.happy/terraform/envs/rdev/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable aws_account_id {
+  type        = string
+  description = "AWS account ID to apply changes to"
+  default     = ""
```

Comment:
are defaults required or desirable?

Reply:
They're not required, and they're desirable for situations where we have a very common input value that we don't want to redefine multiple times.